<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1721</width>
    <height>979</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(202, 202, 202);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelOriginalImage">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>501</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>231</width>
      <height>831</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Image Process</string>
    </property>
    <widget class="QPushButton" name="btnLoadCitra">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>LOAD</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnReset">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>800</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>RESET</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCrop">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>CROP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSave">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>800</y>
       <width>101</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>SAVE</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>181</width>
       <height>161</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Citra Slider</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Brightness</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSlider" name="trackbarBrightness">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Contrast</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSlider" name="trackbarContrast">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>120</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btnGrayscale">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>211</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>GRAYSCALE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSmooth">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>450</y>
       <width>211</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>SMOOTH</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDetect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>510</y>
       <width>211</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>DETECT</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelInput">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>500</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(184, 184, 184);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelHasil">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>60</y>
      <width>501</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(184, 184, 184);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelCroppedImage">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>70</y>
      <width>291</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(184, 184, 184);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>40</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cropped Image</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>210</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Plate Detected</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelDeteksiPlatHasil">
    <property name="geometry">
     <rect>
      <x>1410</x>
      <y>240</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelDetect">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>500</y>
      <width>501</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(184, 184, 184);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelDetect2">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>500</y>
      <width>501</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(184, 184, 184);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelEdgeImage">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>470</y>
      <width>501</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelTopContourImage">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>470</y>
      <width>501</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>890</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Direktori Citra :</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDirektori">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>890</y>
      <width>741</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>1570</x>
      <y>900</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>©2022 Fahmi</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelGrayscaleImage">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>0</y>
      <width>501</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelSmoothenedImage">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>30</y>
      <width>491</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>labelInput</zorder>
   <zorder>groupBox</zorder>
   <zorder>labelHasil</zorder>
   <zorder>labelCroppedImage</zorder>
   <zorder>label_12</zorder>
   <zorder>label_13</zorder>
   <zorder>labelDeteksiPlatHasil</zorder>
   <zorder>labelDetect</zorder>
   <zorder>labelDetect2</zorder>
   <zorder>labelOriginalImage</zorder>
   <zorder>labelEdgeImage</zorder>
   <zorder>labelTopContourImage</zorder>
   <zorder>label_8</zorder>
   <zorder>labelDirektori</zorder>
   <zorder>label_9</zorder>
   <zorder>labelSmoothenedImage</zorder>
   <zorder>labelGrayscaleImage</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1721</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGeometry">
    <property name="title">
     <string>Geometry</string>
    </property>
    <addaction name="actionTranslation"/>
    <addaction name="actionTranspose"/>
   </widget>
   <widget class="QMenu" name="menuSpatial">
    <property name="title">
     <string>Spatial</string>
    </property>
    <widget class="QMenu" name="menuConvolution">
     <property name="title">
      <string>Convolution</string>
     </property>
     <addaction name="actionConvKernel_I"/>
     <addaction name="actionConvKernel_II"/>
    </widget>
    <addaction name="menuConvolution"/>
    <addaction name="action2D_Convolution"/>
    <addaction name="actionMedianFiltering"/>
    <addaction name="actionGaussian"/>
    <addaction name="actionSharpening_Laplace"/>
   </widget>
   <widget class="QMenu" name="menuHistogram">
    <property name="title">
     <string>Histogram</string>
    </property>
    <addaction name="actionHGrayscale"/>
    <addaction name="actionHRGB"/>
    <addaction name="actionHEqualization"/>
   </widget>
   <addaction name="menuGeometry"/>
   <addaction name="menuSpatial"/>
   <addaction name="menuHistogram"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionTranslation">
   <property name="text">
    <string>Translation</string>
   </property>
  </action>
  <action name="actionMean_Filtering">
   <property name="text">
    <string>Mean Filtering</string>
   </property>
  </action>
  <action name="actionMedianFiltering">
   <property name="text">
    <string>Median Filtering</string>
   </property>
  </action>
  <action name="actionGaussian">
   <property name="text">
    <string>Gaussian</string>
   </property>
  </action>
  <action name="actionHGrayscale">
   <property name="text">
    <string>Grayscale</string>
   </property>
  </action>
  <action name="actionHRGB">
   <property name="text">
    <string>RGB</string>
   </property>
  </action>
  <action name="actionHEqualization">
   <property name="text">
    <string>Equalization</string>
   </property>
  </action>
  <action name="actionConvKernel_I">
   <property name="text">
    <string>Kernel I</string>
   </property>
  </action>
  <action name="actionConvKernel_II">
   <property name="text">
    <string>Kernel II</string>
   </property>
  </action>
  <action name="action2D_Convolution">
   <property name="text">
    <string>2D Convolution</string>
   </property>
  </action>
  <action name="actionSharpening_Laplace">
   <property name="text">
    <string>Sharpening Laplace</string>
   </property>
  </action>
  <action name="actionTranspose">
   <property name="text">
    <string>Transpose</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
